moshi/src/main/java/com/squareup/moshi/JsonUtf8Writer.java

```diff
@@ -358,15 +358,14 @@ public void close() throws IOException {
    * and escapes those characters that require it.
    */
   static void string(BufferedSink sink, String value) throws IOException {
-    String[] replacements = REPLACEMENT_CHARS;
```

Comment:
This is a performance enhancement in very, very inner-loop code which caches the static field lookup in a local. Please revert.